README.md

```diff
@@ -0,0 +1,82 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Pokemon Training Card Game
+
+> Browse throw 250 pokemon cards and get info like attacks, number of pokedex, value of the cards and pokemon type.
+
+<img src="https://github.com/AlexRS90/pokemon-cardgame/blob/mobile-app/ssHomePage.jpg" width="300" />
+<img src="https://github.com/AlexRS90/pokemon-cardgame/blob/mobile-app/ssDetailPage.jpg" width="300" />
+
+### Milestone 1:
+- Followed Gitflow
+- Created work-branch
+- Created a new react-app
+- Configured components
+- Installed Redux and dependencies
+  - Followed ducks pattern 🦆
+- Added Home Page component 🏠
+- Templated design followed 🎨
+  - Added styles 🧑‍🎨
+- Get data from store
+- Added Details component 🕵️
+- Showed info from pokemon selected
+- Followed template design
+- Added Navbar component 🌐
+  - Added back button ◀️
+  - Added drop down list 📃
+  - Filtered Pokemon by type 🔥 🌊 👁️ 🐉 🦾 🌱
+- Added dynamic routing for each single pokemon
+- Added unit test
+- Deployed web app
+- Recorded video presentation
+
+## Built With
+
+- React & Redux
+- Visual Studio Code
+- Linters
+- Deployed in Heroku
+
+## Live demo
+
+[PokemonTCG-SPA](https://pokemontcgcapstone.herokuapp.com/)
+
+## Set up locally
+
+- Open your terminal and run the following command: <code>git clone https://github.com/AlexRS90/math-magicians.git</code>
+- Run the command <code>npm install</code>
+- Run the command <code>npm start</code>
+
+Once you completed this steps you should see the app running at: http://localhost:3000/ in your browser.
+
+## Author 👤
+
+👤 **Alejandro Ramos**
+
+- GitHub: [@AlexRS90](https://github.com/AlexRS90)
+- Twitter: [@AlejandroRBenji](https://twitter.com/AlejandroRBenji)
+- LinkedIn: [@AlexRS90](https://www.linkedin.com/in/AlexRS90/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/AlexRS90/math-magicians/issues)
+
+## Original Design:
+
+Nelson Sakwa [Behance Profile](https://www.behance.net/sakwadesignstudio)
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
[OPTIONAL] If we are not doing any acknowledgments I recommend you deleting this section to keep the README clean, updated, and professional

Reply:
Thanks for the advice I'll update it! 😃